### PR TITLE
sidebar badges styling

### LINKED ISSUE #137: Sidebar badges styling
**Is your feature request related to a problem? Please describe.**
This will reorganize the badge layouts

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Style the individual badge layout
2. Add descriptions for each badge

## Purpose
More pizzazz to the badge feature

Closes #137

### CODE REVIEW

File: pokeapp/src/styling/components/_badge-sidebar.scss [hunk 1]
```diff
@@ -49,53 +49,100 @@
...
+
+  .progress {
+    height: 100%;
+    border-radius: 10px;
+    background-color: $secondary-color;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    overflow: hidden;
+  }
```
Comment (krferrell) [inline pokeapp/src/styling/components/_badge-sidebar.scss:143]: The custom progress bars look great.

File: pokeapp/src/components/badges/bagdeIcon.js [hunk 2]
```diff
@@ -77,6 +77,7 @@ export const allBadges = [
       squirtleOwned: false,
       charmanderOwned: false,
     },
+    description: "You can achieve this badge by collecting the starting 3"
```
Comment (Lap343) [inline pokeapp/src/components/badges/bagdeIcon.js:80]: Achievements nice!

**Review by kmausolf — COMMENTED** [review]
I love the descriptions.  It's a nice touch!

**Review by Lap343 — APPROVED** [review]
Nice job with the code. Love the progress bar and of course the achievements

**Review by krferrell — APPROVED** [review]
Nice work on this styling!

**Pull request merged**